.tekton/pipeline-service-static-code-analysis.yaml

```diff
@@ -118,7 +118,8 @@ spec:
               image: quay.io/redhat-pipeline-service/static-checks:$(params.target_branch)
               script: |
                 #!/usr/bin/env bash
-                /opt/static-checks/bin/checkov.sh --workspace_dir $(workspaces.source.path)
+                # once this PR merges, we can switch back to /opt/static-checks/bin/checkov.sh but still leverage config from the PR
+                $(workspaces.source.path)/ci/images/static-checks/content/bin/checkov.sh --workspace_dir $(workspaces.source.path) --config-file $(workspaces.source.path)/ci/images/static-checks/content/config/checkov.yaml
```

Comment:
I understand this is counter to what is the motivation behind building the various CI images, but I'm just validating the diagnosis around still not being able to change CI checks and have them run in the PRs themselves.

Reply:
I am not sure I agree with your comment. The CI image main purpose is to guarantee that the dependencies are installed.

The change you are proposing ensures that a change introduced to either the config or the script is validated properly before being committed.

Therefore, I would remove the comment and keep your solution long term.

Is there an impact I might have overlooked?

Reply:
update pushed @Roming22